cmd/skywire-cli/commands/hv/hvpk.go

```diff
@@ -0,0 +1,31 @@
+package clihv
+
+import (
+	"fmt"
+
+	"github.com/bitfield/script"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	ipaddr, err := script.Exec(`bash -c "_gateway=$(ip route show | grep -i 'default via'| awk '{print $3 }'); _ip=${_gateway%.*}.2; printf ${_ip}"`).String()
```

Comment:
Either camelcase or just use `ip`.

Reply:
Also, is there a need to actually get the IP? Skybian has preconfigured Ips.

Reply:
changed / removed / re-implemnted